app-forensics/magicrescue/magicrescue-1.1.10.ebuild

```diff
@@ -0,0 +1,35 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit eutils toolchain-funcs
```

Comment:
I don't see eutils being used in here, it can be dropped.

Reply:
done